Doc/library/random.rst

```diff
@@ -208,6 +208,9 @@ Functions for sequences
    generated.  For example, a sequence of length 2080 is the largest that
    can fit within the period of the Mersenne Twister random number generator.
 
+   .. deprecated:: 3.9
+      In the future, the optional argument *random* will be removed.
```

Comment:
It looks like the `shuffle()` function is deprecated. It should be specified that the argument is deprecated.

You can also use the `deprecated-removed` directive to specify the version of removing.